src/main/java/com/wavefront/opentracing/reporting/WavefrontSpanReporter.java

```diff
@@ -91,12 +170,20 @@ public void report(WavefrontSpan span) {
           span.getDurationMicroseconds() / 1000, source, ctx.getTraceId(), ctx.getSpanId(),
           parents, follows, span.getTagsAsList(), null);
     } catch (IOException e) {
-      if (logger.isLoggable(Level.FINER)) {
-        logger.finer("Dropping span: " + span);
+      if (isLoggable()) {
+        logger.log(Level.WARNING, "error reporting span: " + span, e);
+      }
+      if (metricsReporter != null) {
+        reportErrors.inc();
+        spansDropped.inc();
       }
     }
   }
 
+  private boolean isLoggable() {
```

Comment:
maybe rename to doLog() to avoid confusion with the name
logger.isLoggable()

Reply:
or shouldLog()